components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/listener/ApplicationMgtListener.java

```diff
@@ -117,6 +117,20 @@ public boolean doPreDeleteApplication(String applicationName, String tenantDomai
     public boolean doPostDeleteApplication(String applicationName, String tenantDomain, String userName)
             throws IdentityApplicationManagementException;
 
+    /**
+     * Define any additional actions after deleting an application
+     *
+     * @param serviceProvider
+     * @param tenantDomain
+     * @param userName
+     * @return
```

Comment:
Complete the java doc comment

Reply:
https://github.com/wso2/carbon-identity-framework/pull/2496/commits/d281a949399c81596e58a81897fdd41f5626ecb1